debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleValueConverters.java

```diff
@@ -240,7 +240,13 @@ protected Object convertString(Column column, Field fieldDefn, Object data) {
             return ((CHAR) data).stringValue();
         }
         if (data instanceof Clob) {
-            return ((Clob) data).toString();
+            try {
+                Clob clob = (Clob) data;
+                return clob.getSubString(1, (int) clob.length());
```

Comment:
Can you add a comment why the substring-ing is needed?

Reply:
Sure, the call to `#toString()` returned the Java object's hash representation of the value and not the actual contents of the clob column.  We can either use this method to obtain the full value or we use `#getCharacterStream()` to obtain a `Reader` to do the same.  Do you have a preference either way?

Reply:
No preference really; what you do here seems easier. I was curious though why does the first character have to be skipped over. Can you put a cmment into the source code explaining that? Also, can there be an empty CLOB ("")? If so, that'd need handling.

Reply:
The `java.sql.Clob` interface specifies that the first character is at position 1 and that if you specify a position 0 or a negative length that a `SQLException` is thrown.  It seems `#getSubstring(1, 0)` is a valid operation for an empty `Clob` data type.

Reply:
Added a note in the code.

Reply:
Ok, thx, that helps!